cppcache/test/LoggingTest.cpp

```diff
@@ -315,18 +315,14 @@ TEST_F(LoggingTest, logInit) {
   apache::geode::client::Log::close();
   boost::filesystem::remove("LoggingTest (#).log");
 
-  // Init with invalid filename
+#ifdef WIN32
```

Comment:
Strikes me that this should be its own test.

Reply:
Moved file/disk space init and filename init tests to their own test cases.